modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -26,6 +26,13 @@ Compaction policy may be applied to a cluster or to an individual topic. If both
 * `compact`: This triggers only cleanup of messages with multiple versions. A message that represents the only version for a given key is not deleted.
 * `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
 
+== Tombstone marker removal
+
+When all messages for a given key are deleted, the topic partition keeps one message, called a tombstone, that contains the key, and a null value. A tombstone indicates to clients that the messages have been deleted. How long Redpanda keeps these tombstones depends on the topic cleanup policy:
```

Comment:
This is not entirely accurate to what a tombstone record is.

A tombstone record is a record _explicitly_ produced by a Kafka client with a given key and the value `null`. 
Tombstone records are not generated by a broker or by regularly scheduled garbage collection.
It itself is the "signal" that records with the same key prior to it in the log should be deleted.

Tombstones are removed either by:

1. Typical garbage collection in a `compact,delete` cleanup policy topic per `delete` retention configuration.
2. In a `compact` only topic, tombstones must be "seen" at least twice by the compaction scheduler before they are removed. This is in order to avoid races between consumers and tombstone removal. The value `delete.retention.ms` sets the bound on the time a consumer has for reading from the log before they see an incomplete view (due to tombstone removal). See Confluent's description of `delete.retention.ms` [here](https://docs.confluent.io/platform/current/installation/configuration/topic-configs.html#delete-retention-ms), and another [helpful resource here](https://www.naleid.com/2023/07/30/understanding-kafka-compaction.html#how-do-i-delete-all-events-sharing-a-key-on-a-compacted-topic).